src/react/ui-elements/CopyARNButton.jsx

```diff
@@ -0,0 +1,55 @@
+//@flow
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { Tooltip } from '@scality/core-ui';
+import { Button } from '@scality/core-ui/dist/next';
+import { spacing } from '@scality/core-ui/dist/style/theme';
+
+const InlineButton = styled(Button)`
+  height: ${spacing.sp24};
+  margin-left: ${spacing.sp16};
+`;
+const IconSuccess = styled.i`
+  color: ${props => props.theme.brand.statusHealthy};
+`;
+
+const CopyARNButton = ({ text }: { text: string }) => {
+  const [copySuccess, setCopySuccess] = useState(false);
+
+  useEffect(() => {
+    const timer = setTimeout(() => {
+      setCopySuccess(false);
+    }, 2000);
+    return () => clearTimeout(timer);
+  }, [copySuccess]);
+
+  const copyToClipboard = () => {
+    if (!navigator || !navigator.clipboard) {
+      return;
+    }
+    navigator.clipboard.writeText(text);
+    setCopySuccess(true);
+  };
+
```

Comment:
The logic seems to be duplicated with the one of the Clipboard component. I vote for a refactoring of the existing component such that:
 - we create a custom hook `useClipboard` defined as :
```
type useClipboard = () => {copy: (text: string) => void, status: 'idle' | 'success' | 'unsupported'} 
``` 
 - we create a generic component with a children props wrapping this children with the tooltip mentioning the success of the copy
 - we change the existing clipboard component to use this generic one
 - we change this implementation with the generic component
-